staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1140,7 +1141,7 @@ func (e *Store) DeleteCollection(ctx context.Context, deleteValidation rest.Vali
 					return
 				}
 				if _, _, err := e.Delete(ctx, accessor.GetName(), deleteValidation, options); err != nil && !apierrors.IsNotFound(err) {
-					klog.V(4).Infof("Delete %s in DeleteCollection failed: %v", accessor.GetName(), err)
+					klog.V(4).InfoS("Delete object in DeleteCollection failed", "object", klog.KObj(accessor), "err", err)
```

Comment:
`klog.KObj(accessor)`  translates to the name?

Reply:
before:
I0126 15:29:45.366832   27464 store.go:1146] Delete foo in DeleteCollection failed: this is for test
after:
I0126 15:29:45.366850   27464 store.go:1147] "Deleted object in DeleteCollection failed" object="test/foo" err="this is for test"